kthfsdv-plotting/main.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.animation import FuncAnimation
+import matplotlib
+from matplotlib import pyplot as plt
+from matplotlib.widgets import Button, TextBox
+import pandas as pd 
+
+class Exc2():
+	def __init__(self, tmin, tmax, updateTime):
+		self.tmin = tmin
+		self.tmax = tmax
+		self.updateTime = updateTime
+		self.H = []
+		self.T = []
+		self.pause = False
+		self.saveFolder ="data"
+
+	def h_t(self, t):
+		return 3*np.pi * np.exp(-self.lambdafunc(t))
+
+	def lambdafunc(self, t):
+		return 5*np.sin(2*np.pi*t)
```

Comment:
This may be just be written as
```suggestion
	def h_t(self, t):
		return 3*np.pi * np.exp(-5*np.sin(2*np.pi*t))
```

Reply:
I think the reason Jón wrote in this way because we posted the intermediate function on the training session website. So, he created the lambda function as a middleware.